sidechain/consensus/aura/src/block_importer.rs

```diff
@@ -229,24 +229,6 @@ impl<
 		&self.ocall_api
 	}
 
-	fn handle_import_error(
-		&self,
-		signed_sidechain_block: &SignedSidechainBlock,
-		error: ConsensusError,
-	) -> Result<(), ConsensusError> {
-		match error {
-			ConsensusError::BlockAncestryMismatch(_last_imported_number, ref msg) => {
-				//FIXME: We should trigger block production suspension (TBD) here.
-				warn!("Could not import block {:?} due to: {:?}.", signed_sidechain_block, msg);
-			},
-			// Some erros, such as invalid author, should be ignored because we do not want
-			// to provide possible attack vectors.
-			_ => warn!("Ignoring import error of block {:?}.", signed_sidechain_block),
-		}
-
-		Err(error)
-	}
```

Comment:
Removed this part again, handling the import error is done in the 'parent' of the block importer. Otherwise the block importer would need to call itself, which makes this struct even more complex and is not good design IMO.